arelle/plugin/validate/HMRC/__init__.py

```diff
@@ -61,86 +102,26 @@
 
 genericDimensionValidation = {
```

Comment:
Pre-existing nit: similar to constants above, this would be neater as UPPER_SNAKE_CASE